### PR TITLE
gh-102304: Fix 2 New Stable ABI Functions

### PR DESCRIPTION
**Pull request by ericsnowcurrently**
These were missed.

* Issue: gh-102304

### CODE REVIEW

**Review by Yhg1s — APPROVED** [review]

**Comment by Yhg1s** [comment]
I expect you'll want to backport this to 3.12?

**Comment by vstinner** [comment]
A different fix is to always implement Py_INCREF() and Py_DECREF() as opaque function calls when Py_DEBUG macro is defined. It may be safer in terms of stable ABI, no?

File: Misc/stable_abi.toml [hunk 1]
```diff
@@ -2406,3 +2406,8 @@
     added = '3.12'
 [const.Py_TPFLAGS_ITEMS_AT_END]
     added = '3.12'
+
+[function._Py_IncRefTotal_DO_NOT_USE_THIS]
+    added = '3.12'
```
Comment (Yhg1s) [inline Misc/stable_abi.toml:2411]:
```suggestion
    added = '3.12'
    abi_only = true
```

File: Misc/stable_abi.toml [hunk 2]
```diff
@@ -2406,3 +2406,8 @@
...
+[function._Py_DecRefTotal_DO_NOT_USE_THIS]
+    added = '3.12'
```
Comment (Yhg1s) [inline Misc/stable_abi.toml:2415]:
```suggestion
    added = '3.12'
    abi_only = true
```

**Review by Yhg1s — APPROVED** [review]

**Comment by vstinner** [comment]
> A different fix is to always implement Py_INCREF() and Py_DECREF() as opaque function calls when Py_DEBUG macro is defined. It may be safer in terms of stable ABI, no?

I created PR #104763 to implement this idea.

**Comment by ericsnowcurrently** [comment]
Hmm, I had not meant to merge this before checking with @vstinner about gh-104763.  I'm not sure how that happened accidentally. 😟

That said, I was already leaning toward merging this and then letting gh-104763 roll it back if applicable.  The most important thing for me is that we get the per-interpreter reftotal working for 3.12+.  So I'll go ahead with the backport at this point.

**Comment by vstinner** [comment]
I'm not convinced that ``_DO_NOT_USE_THIS`` suffix is really needed. We have tons of private functions with ``_Py`` prefix that must not be called. If people use them, they are already on their own. For me, this long ``_DO_NOT_USE_THIS`` suffix is just a burden, for example if you have to manually type the full name in a debugger or a profiler to look for them.

**Comment by ericsnowcurrently** [comment]
I'm fine with changing the name, but I'd rather gh-104763 make it irrelevant. 😄

**Pull request merged**